Formula/openmodelica.rb

```diff
@@ -0,0 +1,94 @@
+class Openmodelica < Formula
+  desc "Open-source modeling and simulation tool"
+  homepage "https://openmodelica.org/"
+  # GitHub's archives lack submodules, must pull:
+  url "https://github.com/OpenModelica/OpenModelica.git",
+    revision: "11fcab4f2d6895f2db073572b2bff1a43177313f", tag: "v1.16.5"
+  license "GPL-3.0-only"
+  head "https://github.com/OpenModelica/OpenModelica.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build # for gfortran
+  depends_on "gnu-sed" => :build
+  depends_on "libtool" => :build
+  depends_on "openjdk" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "gettext"
+  depends_on "hdf5"
+  depends_on "hwloc"
+  depends_on "lp_solve"
+  depends_on "omniorb"
+  depends_on "openblas"
+  depends_on "qt"
+  depends_on "readline"
+  depends_on "sundials"
+
+  uses_from_macos "curl"
+  uses_from_macos "expat"
+  uses_from_macos "ncurses"
+
+  # fix omplot, omnotebook, & omoptim until next release (see OpenModelica #7240)
+  patch :DATA
+
+  def install
+    ENV["LDFLAGS"] = "-L#{Formula["openblas"].opt_lib}"
+    ENV["CPPFLAGS"] = "-I#{Formula["openblas"].opt_include}"
```

Comment:
Have you tried building without these two lines? The compiler shims should already be inserting these flags by default.

Reply:
Works.  I think I still need `ENV.append_to_cflags "-I#{MacOS.sdk_path_if_needed}/usr/include/ffi"` for `uses_from_macos "libffi"` tho (recent HEAD commit removed 3rd party `ffi`).

Reply:
It's not building, so you may need actually need Homebrew `libffi` rather than the built-in one. We can ignore the audit failure suggesting you do `uses_from_macos` if that's what's causing the build failure.